src/Pomelo.EntityFrameworkCore.MySql.Design/Internal/MySqlDatabaseModelFactory.cs

```diff
@@ -27,7 +27,7 @@ public class MySqlDatabaseModelFactory : IInternalDatabaseModelFactory
         public MySqlDatabaseModelFactory(/* [NotNull] */ ILoggerFactory loggerFactory)
         {
             // Check.NotNull(loggerFactory, nameof(loggerFactory));
-
+            loggerFactory.AddConsole();
```

Comment:
I can, but when there are issues with tables (like missing PK) or fields (data type mapping) then the end user will no idea what went wrong (or what was skipped).  I used this successfully to fix problems that I encountered.  If you still feel that this is unnecessary, I will remove it.